adjustments/develop : styling

KianMiridoozini opened the pull request:
Most of the changes consist  styling and UX enhancements

Styling (focus on mobile displays but also some general desktop optimizations):
- Profile
- Profile edit form
- Admin dashboard

### Fixed
checkout  "adjustments/develop : updated"  commit for more info

Also merged with latest develop

KianMiridoozini committed:
adjustments/develop

Most of the changes include styling and UX enhancements

Styling (focus on mobile displays):
- Profile
- Profile edit form
- Admin dashboard

Success message color

Profile edit form behaviour

jayb2302 commented on app/views/user/form.php:
```diff
@@ -67,7 +72,4 @@
         </div>
         <button type="submit" class="btn btn-success">Save Changes</button>
     </form>
```
ooowww I had a very good argument for keeping it down

jayb2302 commented on app/views/user/profile.php:
```diff
@@ -24,9 +24,21 @@
+                            <button type="submit" class="btn btn-primary follow">Follow</button>
+                        </form>
+                    <?php endif; ?>
+                <?php endif; ?>
```
Why did you move it to the top again ?

KianMiridoozini replied:
Makes more sense as the button feels so out of place elsewhere.

You'll see when you see the changes

jayb2302 commented on app/views/user/profile.php:
```diff
@@ -38,19 +50,9 @@ class="profile-img">
-                <?php endif; ?>
                 <div class="follow-stats">
                     <small>Following: <?= htmlspecialchars($userProfile->getFollowingCount() ?? 0); ?></small>
+                    <i>|</i>
```
what's this i tag doing?

KianMiridoozini replied:
Separating the following and follewers by |

Following:12 | Followers:34

jayb2302 replied:
it doesnt need to be in a icon tag

KianMiridoozini replied:
I'm blind, I thought you wrote "what is this | doing". totally misread it
Fixed: I have changed it do divs instead. Yes it doesn't need that either, but for the sake of it not blending with other vertical lines in VS Code, I decided to keep it in div tags

jayb2302 commented on public/assets/css/components/user-management.css:
```diff
@@ -90,3 +90,106 @@
+  .update-status-form .button:hover {
+    background: var(--button-hover);
+  }
+}
```
no nesting?

KianMiridoozini replied:
It was 4 am, the nesting was being a pain, so I did it without nesting for now

jayb2302 replied:
What happened to the nesting?

KianMiridoozini replied:
fixed

jayb2302 committed:
style: refactored CSS with nested structure

KianMiridoozini committed:
adjustments/develope : Fixes

styling updated and optimized

Message notification:
messages will now disappear after few second. + styling

KianMiridoozini committed:
fix

KianMiridoozini committed:
fix conflicts

KianMiridoozini committed:
adjustment/develop : updated

Fixed issues regarding nested css in user-management.css

Improved messages

Improved profile page

Improved login page

Improvement suggestions added in card.ss and cocktail.css

KianMiridoozini committed:
Merge branch 'develop' into adjustments/develop

KianMiridoozini committed:
reverted login

jayb2302 committed:
adjustment(develop):
Jupdate

jayb2302 committed:
Merge branch 'adjustments/develop' of https://github.com/jayb2302/Drinx into adjustments/develop

jayb2302 reviewed (approved):
Approved
